data/libs/functions/player.lua

```diff
@@ -554,3 +554,13 @@ function Player:calculateLootFactor(monster)
 		msgSuffix = suffix
 	}
 end
+
+function Player.disableLoginMusic(self)
+	local msg = NetworkMessage()
+	msg:addByte(0x85)
```

Comment:
why?

Reply:
we use this type of code in .cpp

Reply:
This is a behaviour not a customisation, so it should live in CPP.
The desired usage for lua we see is as a script language for customisation and adjustment, using CPP provided code as a plugin/framework not implementing desired behaviours.

If this code was about defining custom login music per player or a specific login situation, then it would make sense to live in lua.

Here you're abusing the onLogin flow to embed behaviour(music disablement is sent via network) that needs to happen. This should live in CPP and at most exported to lua as a function, but I'd argue that it shouldn't even live in the lua onLogin callback, because that's not what is designed for.

I hope this clarifies a bit the thinking around it.

Thanks.